cirq/__init__.py

```diff
@@ -396,6 +396,7 @@
     decompose,
     decompose_once,
     decompose_once_with_qubits,
+    DEFAULT_RESOLVERS,
```

Comment:
This change also exposes resolvers to the public which was not done before?

Reply:
Yes, I did this since I needed this when writing a test and in general people can inject custom resolvers by doing

```python
cirq.read_json(fn, resolvers=[my_resolver] + cirq.DEFAULT_RESOLVERS)
```

I can put this back if others don't like it at the top level